draft-ietf-rtcweb-jsep.xml

```diff
@@ -3565,6 +3566,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         session MUST be restored to the state it was in before
         performing these steps.</t>
 
+        <t>If the description is of type "rollback", follow the
+        processing defined in the <xref target="sec.rollback" /> and
```

Comment:
Also, I think this should be hoisted to the 'Processing a Local Description' section, since you will also need to discuss the states where this is permitted (and the fact that the desc doesn't need to be parsed).